src/Library/demos/Launcher/main.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Launcher",
-  "category": "controls",
-  "description": "Collect the arguments that are needed to open a File or URI",
+  "category": "platform",
+  "description": "Open a File or URI",
```

Comment:
Where was this string from, by the way?

Reply:
https://docs.gtk.org/gtk4/class.UriLauncher.html

Reply:
I mean the string you changed it to?